packages/blockchain-extension/extension/commands/packageSmartContractCommand.ts

```diff
@@ -216,15 +218,65 @@ async function packageJsonNameAndVersion(workspaceDir: vscode.WorkspaceFolder, o
         const message: string = 'Please enter a package name and/or package version into your package.json';
         throw new Error(message);
     }
+
+    const dependencies: any = workspacePackageObj.dependencies;
+    const contractAPI: string = dependencies['fabric-contract-api'];
```

Comment:
Failing to find the contract-api, we should probably check `fabric-shim` - as I guess they could've written a low-level chaincode.